firestore-bigquery-export/scripts/gen-schema-view/src/index.ts

```diff
@@ -41,52 +43,80 @@ const validateInput = (value: any, name: string, regex: RegExp) => {
   return true;
 };
 
+function collect(value, previous) {
+  return previous.concat([value]);
+}
+
+program
+  .name("gen-schema-views")
+  .option("--no-interactive", "Parse all input from command line flags instead of prompting the caller.", false)
+  .option("-p, --project-id <project-id>", "Firebase Project ID for project containing Cloud Firestore database.")
+  .option("-d, --dataset-id <dataset-id>", "The ID of the BigQuery dataset containing a raw Cloud Firestore document changelog.")
+  .option("-c, --collection <collection>", "The Cloud Firestore collection for which schema views should be generated.")
+  .option("-f, --schema-files <file-glob>", "A path in the filesystem specifying a globbed collection of files to read schemas from.", collect, [])
+
 const questions = [
   {
     message: "What is your Firebase project ID?",
-    name: "projectId",
+    name: "project",
     type: "input",
     validate: (value) =>
       validateInput(value, "project ID", FIRESTORE_VALID_CHARACTERS),
   },
   {
     message:
       "What is the ID of the BigQuery dataset the raw changelog lives in? (The dataset and the raw changelog must already exist!)",
-    name: "datasetId",
+    name: "dataset",
     type: "input",
     validate: (value) =>
       validateInput(value, "dataset", BIGQUERY_VALID_CHARACTERS),
   },
   {
     message:
       "What is the name of the Cloud Firestore Collection that you would like to generate a schema view for?",
-    name: "collectionName",
+    name: "collection",
```

Comment:
This is an existing issue, but worth mentioning.

It is not a good idea to couple Firestore `collectionName` with BigQuery `tableName`. For example, https://github.com/firebase/extensions/issues/17.
Let's rename this to `tableNamePrefix`.

We should describe it as the name prefix of BigQuery table generated by the BigQuery export extension.